specification/nhais-adaptor.yaml

```diff
@@ -30,43 +30,50 @@ info:
     - resource names are capitalised and singular, for example the `/Patient` not `/patients`
     - array names are singular, for example `line` not `lines` for address lines
     - data items that are country-specific and thus not included in the FHIR global base resources are usually wrapped in an `extension` object
+    - data items related to an operation rather than the resource are representes as `Parameter`s
+
+    ## Example Requests
+
+    A number of example requests are available as part of the adaptor's user acceptance tests.
+
+    The [Examples](https://github.com/nhsconnect/integration-adaptor-nhais/blob/develop/README.md#examples) section
+    of the README explains these examples in further detail.
 
   contact:
     name: National Integration Adaptors API Support
     url: 'https://github.com/nhsconnect/integration-adaptors'
```

Comment:
wrong repo ;)

Reply:
I left it pointed at the "hub" repo for all of the adaptors. Should this point at the adaptor's repo instead?